client/java/kserve-api/README.md

```diff
@@ -0,0 +1,38 @@
+# KServe API usage samples
+
+OpenVINO Model Server introduced support for [KServe API](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2), including [Triton](https://github.com/triton-inference-server)'s raw format externsion.
+
+This guide shows how to interact with KServe API endpoints on gRPC. It covers following topics:
+- <a href="#grpc-api">GRPC API Examples </a>
+  - <a href="#grpc-server-live">grpc_server_live</a>
+  - <a href="#grpc-server-ready">grpc_server_ready</a>
+  - <a href="#grpc-server-metadata">grpc_server_metadata</a>
+  - <a href="#grpc-model-ready">grpc_model_ready</a>
+  - <a href="#grpc-model-metadata">grpc_model_metadata</a>
+  - <a href="#grpc-model-infer">grpc_infer_dummy</a>
+
+## Before you run the samples
```

Comment:
Please add how you preconfigured (installed) java.

It would be great if you provide
apt install command with exact package version used.